File: frontend/src/types/consensus.ts
// ── 共识/分歧显示类型 ──

export type ConsensusType = 'consensus' | 'disagreement';

export interface ConsensusItemDisplay {
  id: string;
  type: ConsensusType;
  title: string;
  description: string;
  sourceUtteranceIds: string[];
  confidence: number;
  isResolved: boolean;
  roundNum: number;
}

File: backend/app/schemas/consensus.py
"""共识/分歧 Response Schema"""

from pydantic import BaseModel


class ConsensusResponse(BaseModel):
    id: str
    type: str
    title: str
    description: str
    source_utterance_ids: list[str]
    confidence: float
    is_resolved: bool
    round_num: int
